src/gardener.js

```diff
@@ -88,12 +96,16 @@ module.exports = ({
     'depused'
   ]
     .filter(e => skip.indexOf(e) === -1)
-    .reduce((prev, cur) => prev.then(() => {
-      logger.info(`Running: ${chalk.green(cur)}`);
-      return tasks[cur]();
-    }), Promise.resolve())
+    .reduce(
+      (prev, cur) =>
+        prev.then(() => {
```

Comment:
I'm not sure how much I like the `below` option for `implicit-arrow-linebreak`. Discuss tomorrow?

Reply:
That's not an option...

Reply:
https://eslint.org/docs/rules/implicit-arrow-linebreak, is it not?

Reply:
It is in eslint. But that eslint rule is disabled since prettier now takes care of that. And you can't disable prettier rules

Reply:
> can't disable prettier rules

Let's discuss that in person, I'm surprised you're on board :)

Reply:
The idea is that there are no more format discussions. For the better and the worse :)